R/controls.R

```diff
@@ -544,7 +542,7 @@ bscols <- function(..., widths = NA, device = c("xs", "sm", "md", "lg")) {
 
   ui <- tags$div(class = "container-fluid crosstalk-bscols",
     # Counteract knitr pre/code output blocks
-    tags$div(class = "fluid-row",
+    tags$div(class = "row",
```

Comment:
@JohnCoene why has this changed? Does the grid css we're including only support `.row` and not `.fluid-row`?

Reply:
Oh, ha, it seems `fluid-row` is not a BS3 thing and this should've always been `.row`?

Reply:
I have the latest bslib from CRAN on my machine `0.2.5.1`.

`grep -r fluid-row path/to/site-library/bslib/lib/bs-sass/assets/stylesheets/bootstrap/` returns nothing (path found in `@import` obtained `bs_theme(version = 3)`). `fluid-row` is not available to the best of my knowledge, hence I use `row`.